test/fixtures/lines-between-children.expected.scss

```diff
@@ -0,0 +1,29 @@
+.platform-organize {
+    background-image: linear-gradient(-45deg, #ff83b2 0%, #da82d8 49%, #b480ff 100%);
+
```

Comment:
Remove this empty line because it won't be added based on sort-order in this test.